packages/elements/src/datetime-picker/__test__/datetime-picker.default.test.js

```diff
@@ -108,6 +108,10 @@ describe('datetime-picker/DatetimePicker', function () {
       );
       expect(el.format).to.be.equal(customFormat, 'Custom format is not passed');
       expect(el.inputEl.value).to.be.equal('21-04-20 14:58:59', 'Custom format is not applied');
+
```

Comment:
I think it might be better to create a new test case.

Reply:
Updated.